### DISCUSSION TITLE
Exclude selected jdbc statements from tracing/instrumentation

### DISCUSSION THREAD

**Discussion by stephan-strate** [open-telemetry/opentelemetry-java-instrumentation#11518] [Q&A]
I would like to exclude some selected jdbc statements from being traced.
So in general we want to trace our jdbc statements, but some produce a lot of noise and can be ignored. Is there a way to supress them using config?

I did find instructions on how to ignore classes that use `@WithSpan`, but as far as I understood, this does not apply here.

**Comment by laurit** [comment]
> Is there a way to supress them using config?

I don't think you can do this with just config. You could try writing a custom sampler https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/examples/extension/src/main/java/com/example/javaagent/DemoSampler.java
